lib/one.js

```diff
@@ -5,7 +5,7 @@ var files = require('./files');
 var options = require('./args');
 
 module.exports = function one(name, cb){
-    request.get('https://registry.npmjs.org/'+name, {json: true}, function(err, res, body){
+    request.get('https://registry.npmjs.org/'+name.replace('/', '%2F'), {json: true}, function(err, res, body){
```

Comment:
IIRC this is an issue with CouchDB and it's handling of it. The slash needs to be encoded so that `@foo/bar` is pulled as a unique resource and not as the folder `@foo` and the resource of `bar`. It's like a hack to make it serve properly. I believe we got this hack directly from the [`npm client`](https://github.com/npm/npm/blob/master/lib/utils/map-to-registry.js#L16).